tools/code-tools/jcstress.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# shellcheck disable=SC2035,SC2155
+set -euo pipefail
+WORKSPACE=$PWD
+
+function hashArtifacts() {
+  echo "Creating checksums all asmtools*.jar"
+  for file in `ls asmtools*.jar asmtools*-tests.tar.gz` ; do
+      sha256sum $file > $file.sha256sum.txt
+  done
+}
+
+function detectJdks() {
+  jvm_dir="/usr/lib/jvm/"
```

Comment:
This only builds on Linux, I wonder if there's a more O/S independent way to pass the java in

Reply:
For our CI farm we can of course pin to a Linux node if need be

Reply:
Looks like below that Java 11 is required?